CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticDeactivateStateChangesPlugin.java

```diff
@@ -38,7 +38,6 @@ public class AnalyticDeactivateStateChangesPlugin extends SimpleEditPlugin {
 
     
     public AnalyticDeactivateStateChangesPlugin() {
-        // Empty as no parameters are required for this plugin to run
```

Comment:
My apologies. I meant that the entire constructor didn't need to be included (since this is the implicit default)